ui/App.tsx

```diff
@@ -65,6 +65,26 @@ function withSearchParams(Cmp) {
   };
 }
 
+// gets the right runtime navigation item based on the feature WEAVE_GITOPS_FEATURE_GITOPS_RUNTIME
```

Comment:
@opudrovs @jpellizzari i would like to use the feature flag https://github.com/weaveworks/weave-gitops/pull/4162/files#diff-67081178cf02ff87b1326e8b608a6ab4e49a85606346d64607b498fead04b048R29 

to configure the behaviour of the app like 

if WEAVE_GITOPS_FEATURE_GITOPS_RUNTIME enabled use runtime components 
otherwise use flux_runtime components 

can i get some suggestion on how would you do it?